easybuild/easyconfigs/m/MIGRATE-N/MIGRATE-N-4.2.14-foss-2018a.eb

```diff
@@ -0,0 +1,43 @@
+easyblock = 'ConfigureMake'
+
+name = 'MIGRATE-N'
+version = '4.2.14'
+
+homepage = 'http://popgen.sc.fsu.edu/Migrate/Migrate-n.html'
+description = """
+Migrate estimates population parameters, effective population sizes
+and migration rates of n populations, using genetic data.  It 
+uses a coalescent theory approach taking into account history of 
+mutations and uncertainty of the genealogy. 
+"""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+
+sources = ['migrate-%(version)s.src.tar.gz']
+source_urls = [
+    'http://popgen.sc.fsu.edu/currentversions/',
+    'http://popgen.sc.fsu.edu/oldversions/%(version_major)s.x/%(version_major_minor)s',
+    'http://popgen.sc.fsu.edu/oldversions/',
+    'http://popgen.sc.fsu.edu/newversions/',
+]
+
+patches = [
+    'migrate-%(version)s_install.patch',
+    'migrate-%(version)s_xlocale.patch',
+]
```

Comment:
@RvDijk Please include `SHA256` checksums for both `sources` and `patches` (easiest way is to run `eb --inject-checksums`).